pos_analytic_by_config/models/analytic_distribution_model.py

```diff
@@ -12,7 +12,8 @@ class AccountAnalyticDistributionModel(models.Model):
         help="Select a Point of Sale for which the analytic distribution will be used",
     )
 
-    def _get_distribution(self, vals):
+    def _get_distribution(self, new_vals):
+        vals = new_vals.copy()
```

Comment:
It's better to not alter the method signature with other variable names, so please switch `vals` <> `new_vals`.

Reply:
Done, thanks!